### ISSUE TITLE
`Select > Region > Left/Right of playback position` fixes

### ISSUE THREAD

**Issue by dozzzzer** [audacity/audacity#9385]
1. The operation should be renamed to `Select > Region > Left/Right at playback position` (same as in AU3);
2. The logic behind the operation should be kept the same as in AU3. During playback, it sets the start or end of the time selection, respectively, to the time at which the command was executed. When playback is stopped, a dialog appears where you can set arbitrary start and end times for the selection to be made.

**Comment by DavidBailes** [comment]
@dozzzzer . These are important commands/shortcuts for keyboard users.
